packages/SidePanel/src/components/Footer/Footer.js

```diff
@@ -1,32 +1,32 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { footerCSS } from "./Footer.styles";
-import { useFooterOffset } from "../../hooks";
 
 const propTypes = {
   children: PropTypes.node.isRequired,
   height: PropTypes.number,
+  width: PropTypes.number,
   isSticky: PropTypes.bool,
 };
 
 const defaultProps = {
   height: 48,
   isSticky: false,
+  width: 100,
```

Comment:
Can you use the same default prop as the `SidePanel`. This width prop gets passed down from the <SidePanel> component. so it will make sense to use the same PropTypes, and default prop